index.js

```diff
@@ -59,14 +59,48 @@ animalToRemove.remove()
 /************************** EVENTS JS MINI CHALLENGE ******************************/
 
 /***** Deliverable 1 *****/
-function toggleColor(element) {
```

Comment:
This works, but you didn't have to get rid of our helper function here! 

All we need to do is add an event listener to that header. We provide a callback function to be executed when our "click" event gets triggered.
The callback function we can pass in to the event listener is already made for us!. We just need to pass in the event target as an argument to the function.
```
header.addEventListener("click", function(e){
    toggleColor(e.target)
})
```
With ES6 notation
```
header.addEventListener("click", e => toggleColor(e.target))
```